crates/pathfinder/src/state.rs

```diff
@@ -455,7 +455,7 @@ mod tests {
         // assert_eq!(state.eth_log_index, genesis.origin.log_index);
     }
 
-    #[tokio::test]
+    #[tokio::test(flavor = "multi_thread", worker_threads = 1)]
```

Comment:
why `worker_threads = 1`?

Reply:
No specific reason, apart from it making sure that we don't have anything blocking secretly.